File: An2/MPP/mpp-proiect-csharp-DeepSmeag/networking/dto/PersoanaOficiuDTO.cs
using mpp_proiect_csharp_DeepSmeag.models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace networking.dto
{
    [Serializable]
    public class PersoanaOficiuDTO
    {
        PersoanaOficiu persoanaOficiu;
        public PersoanaOficiuDTO()
        {
        }
        public PersoanaOficiuDTO(PersoanaOficiu persoanaOficiu)
        {
            this.persoanaOficiu = persoanaOficiu;
        }

        public PersoanaOficiu getPersoanaOficiu()
        {
            return persoanaOficiu;
        }
    }
}

File: An2/MPP/mpp-proiect-java-DeepSmeag/networkingModule/src/main/java/contests/network/dto/PersoanaOficiuDTO.java
package contests.network.dto;

import contests.model.PersoanaOficiu;

import java.io.Serializable;

public class PersoanaOficiuDTO implements Serializable {
    PersoanaOficiu persoanaOficiu;
    public PersoanaOficiuDTO() {
    }
    public PersoanaOficiuDTO(PersoanaOficiu persoanaOficiu) {
        this.persoanaOficiu = persoanaOficiu;
    }

    public PersoanaOficiu getPersoanaOficiu() {
        return persoanaOficiu;
    }
}
